files/en-us/web/accessibility/aria/roles/tab_role/index.html

```diff
@@ -10,7 +10,7 @@
 ---
 <div>{{Page("/en-US/docs/Web/Accessibility/ARIA/Roles", "Subnav")}}</div>
 
-<p><span class="seoSummary">The ARIA <code>tab</code> role indicates an interactive element inside a <a href="/en-US/docs/Web/Accessibility/ARIA/Roles/Tablist_Role">tablist</a> that, when activated, displays its associated <a href="/en-US/docs/Web/Accessibility/ARIA/Roles/Tabpanel_Role">tabpanel</a>.</span></p>
+<p><span class="seoSummary">The ARIA <code>tab</code> role indicates an interactive element inside a <code>tablist</code> that, when activated, displays its associated <code>tabpanel</code>.</span></p>
```

Comment:
Adding back the tabpanel page has been renamed. Currently it looks like it hasn't published yet https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tabpanel_Role
```suggestion
<p><span class="seoSummary">The ARIA <code>tab</code> role indicates an interactive element inside a <code>tablist</code> that, when activated, displays its associated <a href="/en-US/docs/Web/Accessibility/ARIA/Roles/Tabpanel_Role"><code>tabpanel</code></a>.</span></p>
```

Reply:
> Adding back the tabpanel page has been renamed

Thanks much for fixing that